tests/longevity-1TB-7days.yaml

```diff
@@ -26,7 +26,8 @@ backends: !mux
         gce_instance_type_db: 'n1-highmem-16'
         gce_root_disk_type_db: 'pd-ssd'
         gce_root_disk_size_db: 50
-        gce_n_local_ssd_disk_db: 8
+        gce_n_local_ssd_disk_db: 0
```

Comment:
please revert this change.
(or at least comment it out for now, so we'll have a reference for the usage).
We'll continue with 8 local ssd for now in longevity and will use this support only in some customer cases that require more than 3TB.

Google told us they are going to support up to 9TB in local SSD in the near future (no date yet).